Config/BuildSettings.swift

```diff
@@ -311,6 +311,7 @@ final class BuildSettings: NSObject {
     static var isRoomScreenEnableMessageBubblesByDefault: Bool {
         return self.roomScreenTimelineDefaultStyleIdentifier == .bubble
     }
+    static let roomScreenUseOnlyLatestProfiles: Bool = false
```

Comment:
I think we generally use "Profile" to refer to the whole collection of data (like "User Profile Screen"). I'd probably be more explicit `roomScreenUserOnlyLatestUserAvatarAndName`